sample-package-core/src/main/java/org/nuxeo/sample/core/CustomBulkAction.java

```diff
@@ -0,0 +1,39 @@
+package org.nuxeo.sample.core;
+
+import static org.nuxeo.ecm.core.bulk.BulkServiceImpl.STATUS_STREAM;
+import static org.nuxeo.lib.stream.computation.AbstractComputation.INPUT_1;
+import static org.nuxeo.lib.stream.computation.AbstractComputation.OUTPUT_1;
+
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.bulk.action.computation.AbstractBulkComputation;
+import org.nuxeo.lib.stream.computation.Topology;
+import org.nuxeo.runtime.stream.StreamProcessorTopology;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class CustomBulkAction implements StreamProcessorTopology {
+  public static final String ACTION_NAME = "custombulkaction";
```

Comment:
I would use camelCase for naming action: customBulkAction
or myCustomAction because there is no point in having all bulk actions suffixed with BulkAction, Action is enough

Reply:
and we can have constant with the full name which is prefix  + action name
```    public static final String ACTION_FULL_NAME = "bulk/" + ACTION_NAME;```

Reply:
Modification made